symengine/ntheory_funcs.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SYMENGINE_NTHEORY_FUNCS_H
+#define SYMENGINE_NTHEORY_FUNCS_H
+
+#include <symengine/basic.h>
+#include <symengine/symengine_casts.h>
+#include <symengine/constants.h>
+#include <symengine/functions.h>
+
+namespace SymEngine
+{
+
+class PrimePi : public OneArgFunction
```

Comment:
Should this just be moved to `functions.h`?

Reply:
Good point. I can do it in a future PR.